charts/rancher-monitoring/v0.2.0/values.yaml

```diff
@@ -4,10 +4,10 @@ operator:
   apiGroup: "monitoring.coreos.com"
   image:
     repository: rancher/coreos-prometheus-operator
-    tag: v0.38.1
+    tag: v0.39.0
     prometheusConfigReloader:
       repository: rancher/coreos-prometheus-config-reloader
-      tag: v0.38.1
+      tag: v0.39.0
```

Comment:
https://github.com/rancher/image-mirror/pull/48

Reply:
Blocked on https://github.com/rancher/image-mirror/pull/49, since the original image-mirror PR had a failed Drone build.